### PR TITLE
{chem}[intel/2016a] Abinit 8.0.8

### PR DESCRIPTION
**Pull request by ysagon**

### CODE REVIEW

File: easybuild/easyconfigs/a/ABINIT/ABINIT-8.0.8-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+configopts = "--with-mpi-prefix=$EBROOTIMPI/intel64 --with-trio-flavor='etsf_io+netcdf' --with-dft=flavor='libxc' "
+configopts += '--with-netcdf-incs="-I$EBROOTNETCDF/include -I$EBROOTNETCDFMINFORTRAN/include" '
+configopts += '--with-netcdf-libs="-L$EBROOTNETCDF/lib -lnetcdf -L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff" '
+configopts += '--with-libxc-incs="-I$EBROOTLIBXC/include" --with-libxc-libs="-L$EBROOTLIBXC/lib -lxc" '
+
+dependencies = [
+    ('libxc', '3.0.0'),
+    ('netCDF', '4.4.1'),
+    ('netCDF-Fortran', '4.4.4'),
```
Comment (ocaisa) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-8.0.8-intel-2016a.eb]: `netCDF-Fortran` depends on `netCDF 4.4.0` so you will need to change the dep above to match for Travis to pass it

Comment (boegel) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-8.0.8-intel-2016a.eb]: @ysagon ping?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/91aad859afe8ccb52e723a1ed7daa565 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2085.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/79517370bc64cdcda202e9a7964e9857 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ysagon!

**Pull request merged**